internal/engine/wazevo/backend/isa/amd64/instr.go

```diff
@@ -1480,6 +1514,7 @@ type useKind byte
 const (
 	useKindNone useKind = iota + 1
 	useKindOp1
+	useKindOp1Reg
```

Comment:
As I said aluRmiR can have reg, imm and memory, so this should be generalized regardless of reg or not.
```suggestion
	useKindOp1Op2
```